libchannelserver/ChannelRPCServer.cpp

```diff
@@ -50,10 +50,15 @@
 #include <boost/uuid/uuid_io.hpp>
 #include <string>
 
+#include "libconfig/GlobalConfigure.h"
```

Comment:
plz place these include at the side of others

Reply:
already remove this header file